collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -612,6 +611,11 @@ protected void onActivityResult(int requestCode, int resultCode,
             return;
         }
 
+        if (intent == null) {
+            Timber.i("The intent has a null value for requestCode: " + requestCode);
```

Comment:
Great idea to log the request code for the nullable intent! But currently, it's being set at info level. How about using `Timber.e` here so that someone having the access to the firebase can get to know the exact reason for the intent to be null.